src/macaron/policy_engine/policy_engine.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+"""This module handles invoking the souffle policy engine on a database."""
+
+import os
+import sys
+
+from sqlalchemy import MetaData, create_engine
+
+from macaron.policy_engine.souffle import SouffleError, SouffleWrapper
+from macaron.policy_engine.souffle_code_generator import (
+    SouffleProgram,
+    get_souffle_import_prelude,
+    project_table_to_key,
+    project_with_fk_join,
+)
+from macaron.util import logger
```

Comment:
We should let the `logging` module detect the appropriate logger object for the module.

```suggestion
logger: logging.Logger = logging.getLogger(__name__)
```

Reply:
Addressed in https://github.com/oracle-samples/macaron/pull/46/commits/73c1c81b6bdb6c999112532dab89103685eca48b